vars/sectionDeployToAKS.groovy

```diff
@@ -86,18 +90,33 @@ def call(params) {
           }
 
           onFunctionalTestEnvironment(environment) {
-            stageWithAgent("Functional Test - AKS ${environment}", product) {
+            onPR {
```

Comment:
you don't want this, it's blocking functional tests from running on master
this + removing closing brace and adjusting indenting
```suggestion
```

Reply:
Updated.